client-app/src/examples/news/AppComponent.ts

```diff
@@ -33,7 +39,18 @@ export const AppComponent = hoistCmp({
                 ],
                 appMenuButtonProps: {hideLogoutItem: false}
             }),
-            item: newsPanel()
+            items: [newsErrorPanel({error: newsErrorThrown}), newsPanel({omit: !!newsErrorThrown})]
         });
     }
 });
+
+const newsErrorPanel = hoistCmp.factory(({error}) => {
+    return errorMessage({
```

Comment:
nice pattern here in the local component is to early out on null, avoiding the null checks below.